user-guide/Reference/faq/General_configuration.md

```diff
@@ -60,17 +60,29 @@ The time server will provide the correct time settings for all DMAs.
 
 To turn any computer into a time server, whether it serves as a DMA or not, do the following.
 
+> [!IMPORTANT]
+> Below information applies to Windows Server 2008 R2, Windows 7 and later.
+
 1. Log on locally to the machine that will act as time server, or establish a remote desktop session with it.
 
-1. Open Registry Editor (*Start \> Run... \> regedit*).
+1. Stop the "Windows Time" service via (*Start \> Control Panel \> System and Security \> Administrative Tools \> Services*).
```

Comment:
Don't use parentheses here. If something is an essential part of the sentence structure, it should never be in parentheses.
![image](https://user-images.githubusercontent.com/94605575/201899089-5c746e5c-8ba8-4342-8933-93ca3d27233a.png)

Reply:
Done